config/rbac/role.yaml

```diff
@@ -42,14 +42,6 @@ rules:
   verbs:
   - get
   - list
-  - watch
-- apiGroups:
-  - ""
```

Comment:
why remove these and below ?

Reply:
it is introduced by #235 , however I think that it should be configured in `all_in_one.yaml` instead.

Reply:
should we make the acls in role.yaml auto generated ?